recipes/duckdb/all/conanfile.py

```diff
@@ -89,6 +90,10 @@ def requirements(self):
     def validate(self):
         if self.settings.compiler.cppstd:
             check_min_cppstd(self, self._min_cppstd)
+        # FIXME: drop support MSVC debug shared build
+        if Version(self.version) >= "0.9.2" and \
+            is_msvc(self) and self.options.shared and self.settings.build_type == "Debug":
+            raise ConanInvalidConfiguration(f"{self.ref} does not support MSVC debug shared build")
```

Comment:
Could you report this issue upstream please? It's not very neat to hide such problem behind a ConanInvalidConfiguration.